content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -218,6 +225,8 @@ equal to one. The `nginx` container is granted 2 exclusive CPUs.
 
 #### Static policy options
 
+##### Beta options (visible by default)
+
```

Comment:
Hmm, this is going to conflict with my change, since my option is alpha. Will need to see which one gets merged first and rebase appropriately.